exotations/solvers/exotica_control_rrt_solver/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(exotica_control_rrt_solver)
```

Comment:
Can we make this `exotica_ompl_control_solver` or similar, and then also expose KPIECE? Now that you've done the hard work to force OMPL to work... exposing an additional solver isn't that much more.

Reply:
Can we leave this a separate issue/ PR? It's actually quite a bit of change in the structure of the thing (esp. initializers & exposing the relevant parameters for RRT/KPIECE) and I think I'll leave it for after submission.

Reply:
Nevermind, did it.

However, i still had the `libraryunload` poco exception when having the implementation for the base class in a separate `.cpp` file. This is similar to `AbstractDDPSolver`.

@VladimirIvan The other OMPL solver seems to make this work. I don't know how or why?

Reply:
https://github.com/ipab-slmc/exotica/issues/644